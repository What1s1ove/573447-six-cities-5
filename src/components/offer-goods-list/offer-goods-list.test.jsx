import * as React from 'react';
import {render} from 'enzyme';
import OfferGoodsList from './offer-goods-list';

describe(`OfferGoodsList component`, () => {
  it(`should render OfferGoodsList component`, () => {
    const component = render(<OfferGoodsList goods={[`TV`]} />);

    expect(component).toMatchSnapshot();
  });
});
