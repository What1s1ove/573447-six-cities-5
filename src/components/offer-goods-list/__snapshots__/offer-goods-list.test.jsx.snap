// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferGoodsList component should render OfferGoodsList component 1`] = `
<ul
  class="property__inside-list"
>
  <li
    class="property__inside-item"
  >
    TV
  </li>
</ul>
`;
